admin/test/system/workarea/admin/orders_system_test.rb

```diff
@@ -6,7 +6,12 @@ class OrdersSystemTest < SystemTest
       include Admin::IntegrationTest
 
       def test_attributes
-        order = create_placed_order
+        order = create_placed_order(
+          traffic_referrer: {
+            uri: 'https://foo.bar'
```

Comment:
(minor) In the future `example.com` was set aside specifically for this kinda stuff: https://tools.ietf.org/html/rfc2606